### ISSUE TITLE
Tests on windows fail - fix up path normalisation.

### ISSUE THREAD

**Issue by dazinator** [yuanchuan/find#19]
On windows, the npm tests fail due to mismatch on path seperator direction, i.e:

```
Expected:
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw/tmp-Bq9I2n
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw/tmp-D1uku3
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw/tmp-Hvtz1M
Actual:
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw\tmp-Bq9I2n
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw\tmp-D1uku3
C:\Users\Daz\AppData\Local\Temp\tmp-205447Z6aY3mCqmVw\tmp-Hvtz1M
```

I'm working on a PR at the moment so i will fix them up.

**Comment by dazinator** [comment]
Fixed in PR #21

**Comment by zanawar** [comment]
+1 on this. Discovered this issue myself recently.
